math-scala/src/main/scala/org/apache/mahout/math/drm/package.scala

```diff
@@ -115,6 +121,46 @@ package object drm {
     }
   }
 
+  /**
+   * Convert arbitrarily-keyed matrix to int-keyed matrix. Some algebra will accept only int-numbered
+   * row matrices. So this method is to help.
+   *
+   * @param drmX input to be transcoded
+   * @param computeMap collect `old key -> int key` map to front-end?
+   * @tparam K key type
+   * @return Sequentially keyed matrix + (optionally) map from non-int key to [[Int]] key. If the
+   *         key type is actually Int, then we just return the argument with None for the map,
+   *         regardless of computeMap parameter.
+   */
+  def drm2IntKeyed[K: ClassTag](drmX: DrmLike[K], computeMap: Boolean = false): (DrmLike[Int], Option[DrmLike[K]]) =
```

Comment:
This looks like a pass-through to DistributedEngine; am I reading it right or missing something?